src/StatsBase.jl

```diff
@@ -2,7 +2,7 @@ __precompile__()
 
 module StatsBase
 
-import Base: length, isempty, eltype, values, sum, show, maximum, minimum, extrema
+import Base: length, isempty, eltype, values, sum, show, maximum, minimum, extrema, depwarn
```

Comment:
`import` is for when you override a function. Just use `Base.depwarn` when you call it (adding it to `using` would run the risk of forgetting to remove it once the deprecation is removed).